lib/submit-request.js

```diff
@@ -40,9 +41,51 @@ function SubmitRequest(backend, agent, index, id, op, options) {
   this.snapshot = null;
   this.ops = [];
   this.channels = null;
+  this._fixupOps = [];
 }
 module.exports = SubmitRequest;
 
+SubmitRequest.prototype.$fixup = function(op) {
```

Comment:
I don't know if adding a function to this is the best interface? I don't know what else to suggest: consumers could just push ops onto an array, but it wouldn't perform all the checks I'm performing here, or immediately update `request.op`, etc.

As far as naming goes, I know `$` normally marks internals, but we already have non-prefixed, internal-only methods on this, so I wanted something to set this apart. It feels not crazy; frameworks like `vue` use the `$` symbol for component utility methods.